### PR TITLE
Fabo/sign in with address

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by fedekunze** [comment]
just FYI, https://github.com/luniehq/lunie/pull/2421/files#diff-98371ed3d92632d8364c257c9c4d2326L6 says `Cosmos Voyager`

File: app/src/renderer/components/common/TmSessionExplore.vue [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+          Explore
+        </div>
+        <a @click="$store.commit(`toggleSessionModal`, false)">
+          <i class="material-icons">close</i>
+        </a>
+      </div>
+      <div class="tm-session-main">
+        <tm-form-group field-id="sign-in-name" field-label="Address">
+          <tm-field
+            v-model="address"
```
Comment (fedekunze) [inline app/src/renderer/components/common/TmSessionExplore.vue]: shouldn't this also be validated ?

File: app/src/renderer/components/common/TmSessionWelcome.vue [hunk 2]
```diff
@@ -17,6 +17,15 @@
           subtitle="If you have a Ledger Wallet, choose this option."
           @click.native="setState('hardware')"
         />
+        <li-session
+          v-if="accountExists"
+          id="explore-address"
+          icon="search"
+          title="Sign in with Address"
+          subtitle="If you just want to explore your address
```
Comment (jbibla) [inline app/src/renderer/components/common/TmSessionWelcome.vue]:
```suggestion
          subtitle="If you want to use Lunie with an address
```

File: app/src/renderer/components/common/TmSessionExplore.vue (hunk 1 above)
Comment (faboweb) [reply app/src/renderer/components/common/TmSessionExplore.vue]: it is now 👍

**Pull request merged**